README.md

```diff
@@ -190,7 +190,34 @@ gnag {
 ***NOTE:*** All reporters are enabled by default
 
 - ***enabled*** - easily disable Gnag in specific situations
-- ***failOnError*** - should violations cause the build to fail or just generate a report
+- ***failOnError*** - should violations cause the build to fail or just generate a report; if set to false, you may need to add the following to prevent your build still failing from Android Lint errors:
+
+    <details open>
+    <summary><b>groovy</b></summary>
+
+    ```
+    android {
+        lintOptions {
+            abortOnError false
+        }
+    }
+    ```
+    
+    </details>
+    
+    <details open>
+    <summary><b>kotlin</b></summary>
+
+    ```
+    android {
+        lintOptions {
+            abortOnError = false
```

Comment:
I'm not sure that it matters but if we want to remain consistent with the rest of the Kotlin documentation we might want to change the syntax to `abortOnError(false)`

Reply:
That depends on how `abortOnError` is defined in the plugin; let me check real quick.

Reply:
It's unclear, doing some experiments!

Reply:
From #218 can confirm, the syntax in Kotlin build scripts needs to be:

```java
android {
    lintOptions {
        setAbortOnError(false)
    }
}
```

<img width="395" alt="Screen Shot 2019-04-02 at 3 16 32 PM" src="https://user-images.githubusercontent.com/6463980/55433008-5f90ec80-5562-11e9-8efc-0149ab408d6c.png">
<img width="322" alt="Screen Shot 2019-04-02 at 3 16 44 PM" src="https://user-images.githubusercontent.com/6463980/55433009-5f90ec80-5562-11e9-9de7-f2512e6abe12.png">